src/FixedDataTableBufferedRows.js

```diff
@@ -96,7 +97,11 @@ class FixedDataTableBufferedRows extends React.Component {
       this._staticRowArray.length = rowsToRender.length;
     }
 
-    var baseOffsetTop = props.offsetTop - props.scrollTop;
+    const height  = props.height;
+    // we translate all the rows together using a parent DIV
+    // the problem with this approach is that for HUGE numbers, multiple translates aren't accurate, so
```

Comment:
So the problem is that for huge no:of rows, the container's offset could be something like -1111111111111111111111 (`-1.1111111111111111e+21`). 
Now for the first row in the viewport, the offset will be 1111111111111111111111
(`1.1111111111111111e+21`). But the translates won't cancel each other properly. Infact all children might lie on the same position. I'm guessing this is some float error that comes up.

The fix was to limit the value to a smaller number. So now if the limit is 1000, then first row shall be at 1000.
The row behind it will be at 900, and the one above can be at 1100.

Reply:
What's the reason for dividing by the table height?  Could we use any factor?  Or must it be a factor larger than the viewport, but any larger will do?

Also have you test this with the ownerHeight property.  I don't think that's used in any of our examples, but we may want to add one.

Reply:
No reason at all. Any factor works.
The greater the number, the better the performance since there's less chance of scrolling across values where the buffer indexes change. I'll put a big number that's safe and update the comment.

Now that the number doesn't depend on the table height, I think it's safe. 
We do have a example that works on `ownerHeight` --only seems to be in Beta -- and it's working with the change.